lucille-plugins/lucille-pinecone/pom.xml

```diff
@@ -24,6 +24,12 @@
       <scope>provided</scope>
     </dependency>
 
+    <dependency>
+        <groupId>com.fasterxml.jackson.core</groupId>
+        <artifactId>jackson-databind</artifactId>
+        <version>2.17.0</version>
+    </dependency>
```

Comment:
For some reason, I am unable to create documents by calling Document.create('id') without it throwing error that it is unable to find class ObjectNode. So for temporary measures adding the dependency here.

Reply:
found that this was due to my testing environment not containing the class.

Reply:
Still unable to remove dependency. Github build fails with same error when dependency not included. So far only passes when included or when scope for lucille-core dependency is set to compile.

`<dependency>
      <groupId>com.kmwllc</groupId>
      <artifactId>lucille-core</artifactId>
      <version>${project.version}</version>
      <scope>compile</scope>
    </dependency>
`

Reply:
This is fine to add the dependency. Do not specify the version here because the project as a whole is already importing the jackson bom in lucille-parent.

So it should be

```
    <dependency>
        <groupId>com.fasterxml.jackson.core</groupId>
        <artifactId>jackson-databind</artifactId>
    </dependency>```

And it will always pull in the same version that lucille is using.

Reply:
resolved in [807c2fb](https://github.com/kmwtechnology/lucille/pull/187/commits/807c2fb3b69ce3736420f2909902e90958310386)